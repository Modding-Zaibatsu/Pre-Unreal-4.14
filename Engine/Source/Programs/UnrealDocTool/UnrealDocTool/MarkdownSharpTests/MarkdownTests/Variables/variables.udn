[VAR:Test]
[OBJECT:Topic]
    [PARAM:image]
        ![%TestA%](Editor/editor_topic.png)
    [/PARAM]
    [PARAM:icon]
        ![](%ROOT%/editor_icon.png)(convert:false)
    [/PARAM]
    [PARAM:title]
        %TestA%
    [/PARAM]
    [PARAM:description]
        %TestB%
    [/PARAM]
    [PARAM:path]
        [RELATIVE:Editor]
    [/PARAM]
[/OBJECT]
[/VAR]

[VAR:TestA]
Test A
[/VAR]

[VAR:TestB]
Test B
[/VAR]

%Test%

[VAR:TablePartVar]
| cell A | cell B |
[/VAR]

[VAR:TestVar]
Test Var
[/VAR]

| Table Col A | Table Col B |
| ----------- | ----------- |
%TablePartVar%

Normal paragraph %TestVar%. Normal paragraph %TestVar%. Normal paragraph
%TestVar%. Normal paragraph %TestVar%. Normal paragraph %TestVar%. Normal
paragraph %TestVar%. Normal paragraph %TestVar%. Normal paragraph %TestVar%.
Normal paragraph %TestVar%. Normal paragraph %TestVar%. Normal paragraph
%TestVar%.

    Code block test %TestVar%. Code block test. Code block test. Code block test
    %TestVar%. Code block test. Code block test. Code block test %TestVar%.
    Code block test. Code block test. Code block test %TestVar%. Code block
    test. Code block test.

1. List test %TestVar%. List test %TestVar%. List test %TestVar%. List test
    %TestVar%. List test %TestVar%. List test %TestVar%. List test %TestVar%.
    List test %TestVar%. List test %TestVar%. List test %TestVar%. 

        Code block test %TestVar%. Code block test. Code block test. Code block test
        %TestVar%. Code block test. Code block test. Code block test %TestVar%.
        Code block test. Code block test. Code block test %TestVar%. Code block
        test. Code block test.

1. List test %TestVar%. List test %TestVar%. List test %TestVar%. List test
    %TestVar%. List test %TestVar%. List test %TestVar%. List test %TestVar%.
    List test %TestVar%. List test %TestVar%. List test %TestVar%. 



[VAR:regionClassVar]
testClass
[/VAR]

[REGION:%regionClassVar%]
Test Region
[/REGION]