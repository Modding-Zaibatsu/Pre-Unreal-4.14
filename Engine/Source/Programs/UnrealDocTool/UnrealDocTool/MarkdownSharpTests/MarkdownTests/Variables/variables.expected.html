<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" dir="ltr" lang="en-US">
    <head>
    </head>
    <body>
        <a href="..\Editor\index.html">
            <div class="item">
                <p><img alt="Test A" src="./../../images\Editor\editor_topic.Jpeg" /></p>
                <div class="info">
                    <div class="title">
                        <p>Test A</p>
                    </div>
                    <div class="content">
                        <div class="description">
                            <p>Test B</p>
                        </div>
                    </div>
                </div>
            </div>
        </a>
        <table>
            <caption></caption>
            <col  align="" />
            <col  align="" />
            <thead>
                <tr>
                    <th rowspan="1" colspan="1">
                        Table Col A
                    </th>
                    <th rowspan="1" colspan="1">
                        Table Col B
                    </th>
                </tr>
            </thead>
            <tbody>
                <tr>
                    <td rowspan="1" colspan="1" align="">
                        cell A
                    </td>
                    <td rowspan="1" colspan="1" align="">
                        cell B
                    </td>
                </tr>
            </tbody>
        </table>
        
        <p>
            Normal paragraph Test Var. Normal paragraph Test Var. Normal paragraph
            Test Var. Normal paragraph Test Var. Normal paragraph Test Var. Normal
            paragraph Test Var. Normal paragraph Test Var. Normal paragraph Test Var.
            Normal paragraph Test Var. Normal paragraph Test Var. Normal paragraph
            Test Var.
        </p>

        <pre class="prettyprint"><code>Code block test %TestVar%. Code block test. Code block test. Code block test
%TestVar%. Code block test. Code block test. Code block test %TestVar%.
Code block test. Code block test. Code block test %TestVar%. Code block
test. Code block test.</code></pre>

        <ol>
            <li>
                <p>
                    List test Test Var. List test Test Var. List test Test Var. List test
                    Test Var. List test Test Var. List test Test Var. List test Test Var.
                    List test Test Var. List test Test Var. List test Test Var.
                </p>

                <pre class="prettyprint"><code>Code block test %TestVar%. Code block test. Code block test. Code block test
%TestVar%. Code block test. Code block test. Code block test %TestVar%.
Code block test. Code block test. Code block test %TestVar%. Code block
test. Code block test.</code></pre>
            </li>

            <li>
                <p>
                    List test Test Var. List test Test Var. List test Test Var. List test
                    Test Var. List test Test Var. List test Test Var. List test Test Var.
                    List test Test Var. List test Test Var. List test Test Var.
                </p>
            </li>
        </ol>

        <div class="testClass">
            <p>
                Test Region
            </p>
        </div>
    </body>
</html>
